tutorials/editor/script_editor.rst

```diff
@@ -0,0 +1,409 @@
+  folder. Selecting "Find..." starts with the chosen folder, and includes the file extensions 
+  checked in the filters. The results are shown in the bottom panel with the number of matches 
+  and total number of files found, in the "Search Results" tab. Clicking on a result opens 
+  the file and jumps to the respective line.
+- **Replace in Files...**: Opens a window to search and replace a word with another one within the 
+  found files in the project folder. After clicking "Replace...", you can select in which files to 
+  replace using the "Search Results" tab in the bottom panel by (un)checking them and using 
+  "Replace All" button.
+  
+.. image:: img/editor_ui_script_editor_replaceinfiles.webp
+
+.. warning:: Note that "Replace in Files" operation cannot be undone!
+
+.. tip:: Both the `Find in Files` and `Replace in Files` windows share the "Search..."
+  and "Replace..." buttons. The only difference in the latter window is an additional text
+  field that automatically fills in the search results panel when the "Replace..." button
+  is clicked. The replacement operation is only executed if you click the "Replace All"
+  button in this bottom panel, allowing you to also edit the word to replace later within
+  this panel.
+
+.. image:: img/editor_ui_script_editor_replace_all.webp
+
+- **Contextual Help***: Opens the list of built-in class reference similar to pressing F1
+  on a symbol, or choosing "Lookup Symbol" from the context menu.
+
+**Go To** menu lets you navigate within the code at ease with these options:
+
+.. image:: img/script_editor_icons/text_editor_goto_menu.webp
+
+- **Go to Function...**: Opens the function list to jump to. You can achieve the same result
+  by typing in the filter methods bar in the script panel.
+- **Go to Line...**: Jumps to the written line number in the code editor.
+- **Bookmarks**: Contains actions for the bookmark functionality, which you can use to find
+  your way through your code easier, such as an incomplete section. Bookmarked lines will
+  have a blue bookmark symbol left of the line number.
+
+  - `Toggle Bookmark*`: Adds and removes the bookmark on the line where the caret is. You can
+    also right click on a line to achieve this.
+  - `Remove All Bookmarks`: Removes all bookmarks in the open document.
+  - `Go to Next Bookmark`: Jumps to the next bookmark in the open document.
+  - `Go to Previous Bookmark`: Jumps to the previous bookmark in the open document.
+  - `Bookmarks` menu will also contain the list of bookmarked lines, including their line
+    number and displaying the partial content in that line.
+
+- **Breakpoints**: Breakpoints are helpful while debugging your code. Similar to "Bookmarks"
+  menu, this menu lets you add or remove breakpoints, navigate between them and directly
+  jump to a specific breakpoint. An easy way to add a breakpoint is hovering the blank area
+  left of a line number. The cursor will show a faded red circle. Clicking it will add a
```

Comment:
```suggestion
  jump to a specific breakpoint. An easy way to add a breakpoint is hovering over the blank area
  left of a line number. It will show a faded red circle. Clicking it will add a
```

It is shown in the blank area mentioned, not on the cursor.